packages/testring-cli-config/src/index.ts

```diff
@@ -4,6 +4,8 @@ import { getArguments } from './arguments-parser';
 import { getFileConfig } from './config-file-reader';
 import { defaultConfiguration } from './default-config';
 
+export { defaultConfiguration } from './default-config';
```

Comment:
you already imported module, it's better to make like in
https://github.com/ringcentral/testring/blob/master/packages/testring-test-worker/src/index.ts